tests/integration/Core/Checkout/Customer/SalesChannel/RegisterRouteTest.php

```diff
@@ -1398,6 +1398,141 @@ public function testRegisterWithTooLongPassword(): void
         static::assertSame(':PASSWORD_IS_TOO_LONG', $error['detail']);
     }
 
+    public function testRegistrationWithLongNames(): void
```

Comment:
do we really want to have integration tests for this? any integration test makes the ci little bit slower